include/constants/heal_locations.h

```diff
@@ -24,5 +24,6 @@
 #define HEAL_LOCATION_EVER_GRANDE_CITY_POKEMON_LEAGUE 20
 #define HEAL_LOCATION_SOUTHERN_ISLAND_EXTERIOR 21
 #define HEAL_LOCATION_BATTLE_FRONTIER_OUTSIDE_EAST 22
+#define HEAL_LOCATION_MAX (HEAL_LOCATION_BATTLE_FRONTIER_OUTSIDE_EAST)
```

Comment:
Optional, but I think we generally would use `HEAL_LOCATION_COUNT` with a +1 here